yt/visualization/volume_rendering/off_axis_projection.py

```diff
@@ -132,11 +138,13 @@ def off_axis_projection(data_source, center, normal_vector,
 
         raise_error = False
 
+        # Assure that the field we're trying to off-axis project is a
+        # particle field or a 'gas' field.
```

Comment:
super minor nit: we're not generically checking for particle fields, we're checking if the field type is the SPH particle type or if the field is an alias to an SPH field.